FirstWorkProject/FirstWorkProject/ViewControllers/HomeVC+TableViewMethods.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  HomeVC+TableViewMethods.swift
+//  FirstWorkProject
+//
+//  Created by Satsishur on 14.04.2021.
+//
+
+import UIKit
+
+extension HomeVC: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return dataSource.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! SearchResponceTableViewCell
```

Comment:
Let's use a cell identifier which duplicates the class name - SearchResponceTableViewCell as there can be more than one cell in the project